projects/emissary-ingress/emissary/Makefile

```diff
@@ -0,0 +1,51 @@
+BASE_DIRECTORY:=$(shell git rev-parse --show-toplevel)
+GIT_TAG:=$(shell cat GIT_TAG)
+GOLANG_VERSION?="1.17"
+
+APPMESH_VERSION:=$(shell cat APPMESH_VERSION)
+IMAGE_BUILD_ARGS=APPMESH_VERSION
+
+REPO=emissary
+REPO_OWNER=emissary-ingress
+
+EXCLUDE_FROM_STAGING_BUILDSPEC=true
+BINARY_TARGET_FILES=busyambassador capabilities_wrapper example-envoy-metrics-sink k8sregistryctl kat-client kat-server
+SOURCE_PATTERNS=./cmd/busyambassador ./cmd/capabilities_wrapper ./cmd/example-envoy-metrics-sink ./cmd/k8sregistryctl ./cmd/kat-client ./cmd/kat-server
+GO_MOD_PATHS=.
+
+EMISSARY_IMAGE_COMPONENT=emissary-ingress/emissary
+
+IMAGE_NAMES=emissary
+DOCKERFILE_FOLDER=./docker/linux/emissary-image
+
+# This will registry be changed once the eks-distro-minimal-base-python PR is merged.
+BASE_IMAGE_REPO?=public.ecr.aws/k1e6s8o8
+BASE_IMAGE_NAME?=eks-distro-minimal-base-python
+
+CHART_VERSION=$(shell cat CHART_VERSION)
+
+HELM_PULL=true
```

Comment:
So, this helm chart is not in a git repo we can clone? Using sed to replace all those values, I'm not terribly confident that is going to work. We need to add sourceRegistry to it, the shasums rather than IMAGE_TAG and with those changes Steven is working on some sort of image pull secret, not sure what exactly

Reply:
The helm chart they have in the repo would need a gigantic git patch, but it's possible. It's basically one they template for emissay or ambassador so it's not super clean.

Reply:
Changed it to using the Git patch strategy in [30471f3](https://github.com/aws/eks-anywhere-build-tooling/pull/1059/commits/30471f31dc4e3db0c120b2b4aa43aab0f7eb699f)